controllers/chat/group/changeChannelDetail.js

```diff
@@ -12,6 +14,8 @@ const changeChannelDetail = async (req, res) => {
   const {channel_id, channel_image, channel_name} = req.body;
   const {userId} = req;
 
+  const ownUser = await UsersFunction.findUserById(User, userId);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The user ID is directly passed to the `findUserById` function without any validation. This could lead to potential security vulnerabilities such as SQL Injection. Always validate and sanitize user inputs before using them in database queries.
```diff
- const ownUser = await UsersFunction.findUserById(User, userId);
+ const sanitizedUserId = sanitizeInput(userId); // Add a function to sanitize input
+ const ownUser = await UsersFunction.findUserById(User, sanitizedUserId);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->